.github/workflows/tests.yml

```
@@ -65,7 +67,7 @@ jobs:
             unset CF_WEBSERVICES_FEEDSTOCK_TOKEN
             unset CF_WEBSERVICES_FEEDSTOCK_APP_ID
           fi
-          pytest -v conda_forge_webservices
+          # pytest -v conda_forge_webservices
```

```suggestion
          pytest -v conda_forge_webservices
```

This needs to be put back!